sale_delivery_split_date/readme/ROADMAP.md

```diff
@@ -0,0 +1,2 @@
+- Incompatible with
+  [sale_procurement_group_by_commitment_date](https://github.com/OCA/sale-workflow/tree/12.0/sale_procurement_group_by_commitment_date)
```

Comment:
not that module is called sale_procurement_group_by_requested_date: https://github.com/OCA/sale-workflow/tree/15.0/sale_procurement_group_by_commitment_date

Reply:
Please disregard the comment, I got confused with module names